internal/v2/s2av2_e2e_test.go

```diff
@@ -128,9 +132,33 @@ func startServer(t *testing.T, s2aAddress string, localIdentities []*commonpbv1.
 	return lis.Addr().String()
 }
 
+// startFallbackServer runs a GRPC echo testing server.
```

Comment:
nit: Please mention that it returns the address on which the server is listening.